root_main.tf

```diff
@@ -2,6 +2,11 @@ module "global_parameters" {
   source = "./tdr-configurations/terraform"
 }
 
+module "terraform_config" {
+  source  = "./da-terraform-configurations/"
+  project = var.project
+}
+
 module "iam" {
   source            = "./tdr-terraform-modules/iam"
```

Comment:
Pointing to the `tdr-terraform-modules` repo, should this point to the `da-terraform-modules` instead?

Reply:
This module is only used in this repo so I've moved it out into this project. I guess it can be removed from `tdr-terraform-modules` at some point but every time I do that I break things.